src/convertToObject.js

```diff
@@ -3,16 +3,33 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in [stylesString.js]
+ * (./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayOfStyles = sourceString.replace(/\n/g, '')
+    .split(';')
+    .map(element => element.trim())
+    .filter(element => element !== '')
+    .map(line => {
+      const temp = line.split(':');
+
+      return [temp[0].trim(), temp[1].trim()];
+    });
+  const result = {};
```

Comment:
2. [CODE STYLE] - Avoid variable namings like `array , object arr, obj,` `result`. It will decrease code readability